code/modules/mob/living/silicon/robot/dogborg/dog_sleeper_vr.dm

```diff
@@ -84,7 +84,7 @@
 				update_patient()
 			return
 /* //VORESTATION AI TEMPORARY REMOVAL
-		if(istype(target, /mob/living/simple_mob/mouse)) //Edible mice, dead or alive whatever. Mostly for carcass picking you cruel bastard :v
+		if(istype(target, /mob/living/simple_mob/animal/passive/mouse)) //Edible mice, dead or alive whatever. Mostly for carcass picking you cruel bastard :v
```

Comment:
Can you test to see if that snippet of code would now work with this change? I would like to steadily fix and implement the code commented out due to the AI overhaul

Reply:
boop